projects/dynamicforms/tsconfig.lib.json

```diff
@@ -10,10 +10,7 @@
     "experimentalDecorators": true,
     "importHelpers": true,
     "types": [],
-    "lib": ["es2020", "dom"],
-    "paths": {
-      "@fsa/dynamicforms": ["dist/dynamicforms"]
-    }
+    "lib": ["es2020", "dom"]
```

Comment:
Shouldn't there be some path for build in develop mode? Even if the default is dist, I would say it's a good practice to have it here as well.

Reply:
Path should be defined if we have this path reference in lib. 
For example we have this path in fsastorefrontlib because from there we are referencing dynamicforms, but in dynamicforms there is no reason that we add this path because files from dynamicforms are referenced directly, not over library.

Reply:
That also goes when customer uses dynamicforms without FSA?